### ISSUE TITLE
Enable Google Workspace SMTP server for Feedback API

### ISSUE THREAD

**Issue by donpui** [LeastAuthority/feedback-api#3]
Feedback will be received via API, an email message will be generated and sent to internal feedback@winden.app

For backend server, we need to have access to SMTP server to send email.
There are three options according [Google Support](https://support.google.com/a/answer/176600?hl=en):
- Option 1: Send email with SMTP relay
- Option 2: Send email with the Gmail SMTP server
- Option 3: Send email with the restricted Gmail SMTP server

Basically we need only to send to internal mailbox. It should be sufficient to go with Option 3 without additional authentication only binding to IP. 

Note: Feedback-API service is planned to run as docker container on already existing OVH VPS server.
